db/migrate/20220509110837_add_id_offset_organisations_and_logs.rb

```diff
@@ -0,0 +1,8 @@
+class AddIdOffsetOrganisationsAndLogs < ActiveRecord::Migration[7.0]
+  def up
+    execute "SELECT setval('organisations_id_seq', 100000000)"
+    execute "SELECT setval('case_logs_id_seq', 300000000000)"
```

Comment:
300,000,000,000

The max size of a bigint is 9,223,372,036,854,775,807, so starting at this offset is not a problem.